src/deluge/io/midi/midi_device.cpp

```diff
@@ -62,7 +62,7 @@ void MIDIDevice::dataEntryMessageReceived(ModelStack* modelStack, int32_t channe
 		if (zone >= 0) { // If it *is* MPE-related...
 
 			// If it's on the "main" channel
-			if (channel == 0 || channel == 15) {
+			if (ports[MIDI_DIRECTION_INPUT_TO_DELUGE].isMasterChannel(channel)) {
```

Comment:
channel == 0 || channel == 15 is not a sufficient check to tell the truth of those channels, as main channels, because an MPE zone could be totally disabled